themes/default/content/blog/billing-managers/index.md

```diff
@@ -0,0 +1,66 @@
+---
+title: "Introducing Billing Managers: A new role in the Pulumi Service"
+allow_long_title: true
+
+date: 2022-07-27T09:00:22-07:00
+
+meta_desc: Starting today all Pulumi Organizations can have a Billing Manager, a role designed for your finance function.
+
+meta_image: meta.png
+
+authors:
+    - isabel-suchanek
+    - meagan-cojocar
+
+tags:
+    - features
+    - pulumi-service
+---
+
+We are excited to announce that starting today Pulumi customers can create a Billing Manager role for their organization on the [Pulumi Service](https://app.pulumi.com). We have heard from our customers that having a Billing Manager would solve internal process challenges and are aiming to solve these pain points with the new Billing Manager role.
+
+<!--more-->
+
+## Feature overview
+
+The Billing Manager role gives the user permissions to access and edit only billing information, with no read or write permissions for stacks. This role is for when you need  someone in your Organization to manage billing operations but you do not want them to have any permissions beyond that. Billing Managers can only be invited or added to the Organization by Organization Admins. Once added, the only information a Billing Manager will see is metrics about the Organization and the Billing and Usage page. Learn more about the permissions in the [Billing Manager documentation]({{< relref "/docs/" >}}).
```

Comment:
These are the docs we're adding later right?

Reply:
Yes exactly! I can remove this if we don't get them out at the same time. It would take one of us like an hour max to write them up I think.